app/services/search/postgres/listing.rb

```diff
@@ -0,0 +1,58 @@
+module Search
+  module Postgres
+    class Listing
+      ATTRIBUTES = %i[
+        id
+        body_markdown
+        bumped_at
+        cached_tag_list
+        classified_listing_category_id
+        contact_via_connect
+        expires_at
+        organization_id
+        originally_published_at
+        location
+        processed_html
+        published
+        slug
+        title
+        user_id
+      ].freeze
+
+      DEFAULT_PER_PAGE = 75
+      MAX_PER_PAGE = 150 # to avoid querying too many items, we set a maximum amount for a page
```

Comment:
I just arbitrally chose 2x the `DEFAULT_PER_PAGE` value. I'm open to adjusting this!